dace/cli/sdfv.py

```diff
@@ -36,7 +36,7 @@ def view(sdfg: dace.SDFG, filename: Optional[Union[str, int]] = None):
     """
     # If vscode is open, try to open it inside vscode
     if filename is None:
-        if 'VSCODE_IPC_HOOK_CLI' in os.environ or 'VSCODE_GIT_IPC_HANDLE' in os.environ:
+        if 'VSCODE_IPC_HOOK' in os.environ or 'VSCODE_GIT_IPC_HANDLE' in os.environ:
```

Comment:
I just checked - in my environment (Ubuntu20), VSCode does not set the `VSCODE_IPC_HOOK` variable in the terminal, nor the debug console, weirdly enough. In that case I would request that we test for both `VSCODE_IPC_HOOK` and `VSCODE_IPC_HOOK_CLI`.